regression/validate-trace-xml-schema/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# The Schema tests require a JDK because we use the
+# Java standard libraries builtin schema validation functionality
+find_package(Java COMPONENTS Development)
+find_package(PythonInterp 3.5)
+if(${PYTHONINTERP_FOUND}
+  AND ${Java_FOUND} AND ${Java_VERSION_MAJOR} VERSION_GREATER 8)
+    add_test(NAME validate-trace-xml-schema
+            COMMAND ${PYTHON_EXECUTABLE} check.py $<TARGET_FILE:cbmc>
+            WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR})
+    set_tests_properties(validate-trace-xml-schema
+            PROPERTIES
+            LABELS "CBMC;CORE")
+else()
+    message(WARNING "JDK not found, skipping trace schema tests")
```

Comment:
:question: This stops the tests being configured? I wonder if we should configure a test that always fails? That way, if someone configures it locally, they don't need JDK, but if CI fails on this step, it doesn't silently pass.

Reply:
Not all our CI passes have both Java and Python 3 available.

Reply:
I guess I'd rather we explicitly _skip_ this test on the jobs that don't have the requirements.